tests/e2e/browser/utils.ts

```diff
@@ -0,0 +1,92 @@
+import { test, Page, Locator, expect } from "@playwright/test";
+import { getQuasarMenu } from "../locators";
+import {
+  mockReadFile,
+  mockShowOpenFileDialog,
+  mockShowSaveFileDialog,
+  mockWriteFile,
+} from "./mockUtility";
+import { UnreachableError } from "@/type/utility";
+
+/** UIのロックが解除されるまで待つ */
+export async function waitForUiUnlock(page: Page): Promise<void> {
+  await test.step("UIのロックが解除されるまで待つ", async () => {
+    const addAudioButton = page.getByLabel("テキストを追加");
+    await expect(addAudioButton).toBeEnabled({ timeout: 10 * 1000 });
+  });
+}
+
+/**
+ * ページ内でプロジェクトを読み込む。
+ *
+ * @param projectJson 読み込むプロジェクトファイルの内容
+ */
+export async function loadProject(
+  page: Page,
+  projectJson: string,
+): Promise<void> {
+  await test.step("プロジェクトを読み込む", async () => {
+    const testProjPath = `/tmp/${Date.now()}-testProj.vvproj`;
+    await mockReadFile(page, testProjPath, Buffer.from(projectJson, "utf-8"));
+    await mockShowOpenFileDialog(page, testProjPath);
+    await page.getByRole("button", { name: "ファイル" }).click();
+    await getQuasarMenu(page, "プロジェクトを読み込む").click();
+
+    // TODO: 保存ダイアログに対応する
```

Comment:
これ初見だと何のことかわからないかも。
こうですかね？

```suggestion
    // TODO: 編集中の内容を保存するか問うダイアログに対応する
```